server/report_builder/README.md

```diff
@@ -221,3 +222,103 @@ Next it will fallback to the english translation of the nominated key in the com
 Next it will fallback to the fallback text provided in the report which by default will be in english
 
 If none of the above can be found, report will fail to render.
+
+# Standard Reports
+
+Builds 2.3.0 and above will include standard reports embedded in the binary. The 2.3.0 build includes the following reports:
+
+- Invoice Landscape
+- Item Usage
+- Stock Detail
+- Stock Status
+- Expiring Items
+
+Standard reports are upserted into the database on startup.
+These are built and added from the standard_reports.json file in reports/generated. This json file includes all standard reports, and all versions of each report.
+
+## Building standard reports
+
+Standard reports are built from html, css, and query files a similar way as from the reports repo with the following differences:
+
+Standard reports include a 'manifest.json' file which includes details of how the report is constructed.
+Optional fields in the manifest json are marked as '// optional'
+
+```json
+{
+  // is_custom is specified as false for standard reports and by default true for other reports.
+  // custom reports will override standard reports of the same report code.
+  "is_custom": false,
+  // open mSupply needs to know the report version
+  "version": "2.3.0",
+  // The code is a new field which is the unique identifier of a report group inclusive of all versions of that report.
+  // Each different version of a standard report will have the same code.
+  // The code is required to identify reports of the same version where they will have different ids
+  "code": "item-usage",
+  // report context, see server/repository/src/db_diesel/report_row.rs
+  "context": "REPORT",
+  // optional
+  // Sub context for reports, applicable to Dispensary and Report context, see client/packages/reports/src/ListView/ListView.tsx",
+  // Currently only reports of subcontext 'Expiring' and ' StockAndItems' are displayed
+  "sub_context": "StockAndItems",
+  // Display name of report
+  "name": "Item Usage",
+  // optional
+  "queries": {
+    // optional
+    // graphql file query name with extension
+    "gql": "query.graphql",
+    // optional
+    // vec of sql query file names without extension. report_builder will parse both postgres.sql, sqlite.sql, and agnostic .sql file type extensions.
+    "sql": ["thisMonthConsumption", "lastMonthConsumption"]
+  },
+  // optional
+  "arguments": {
+    // optional
+    // location of schema file json relative to the version dir
+    "schema": "argument_schemas/arguments.json",
+    // optional
+    // location of ui schema file json relative to the version dir
+    "ui": "argument_schemas/arguments_ui.json"
+  }
+}
+```
+
+This manifest.json file takes the place of cli param inserts seen in bash scripts such as upsert.sh in the open-mSupply-reports repo.
+
+Standard reports in the reports dir can be built into the generated json by running
+
+```bash
+./target/debug/remote_server_cli build-standard-reports
```

Comment:
Are they just built or also upserted? Isn't there two commands you added?
Should we also mention how to build the cli?